agent/api/ecsclient/client.go

```diff
@@ -454,7 +454,7 @@ func (client *APIECSClient) buildManagedAgentStateChangePayload(change api.Manag
 	}
 }
 
-func (client *APIECSClient) buildContainerStateChangePayload(change api.ContainerStateChange) *ecs.ContainerStateChange {
+func (client *APIECSClient) buildContainerStateChangePayload(change api.ContainerStateChange, shouldExcludeIPv6PortBinding bool) *ecs.ContainerStateChange {
```

Comment:
is there any reason why ipv6 ports are excluded only in state change calls? 
Wouldn't the ipv6 ports show up in task metadata endpoint otherwise? 
its better to filter it here to avoid any discrepancies:
https://github.com/aws/amazon-ecs-agent/blob/225bc3a556bd2d1759ab27b23f54e7e68086c9f0/agent/engine/docker_task_engine.go#L494

Reply:
Update README file and the description for the changelog to make the purpose/impact of this PR more clear. Thanks!